.github/workflows/test-policies.yml

```diff
@@ -78,5 +78,7 @@ jobs:
         enable-AzPSSession: true 
     - name: Test Azure Policies
       shell: pwsh
+      env:
+        AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
```

Comment:
Thanks for the pull request! One question: Why is passing the Azure Credentials using an environment variable necessary? The login happens in the previous workflow step, so additional login is not required. Thank you.

Reply:
Well, it depends on the way you write your tests. I too have some testcases were I test if the Policy is doing something and therefore, I would like to change the enforcement mode. As I don't want to wait 30' I would like to do a logoff/logon from within the test script.

Reply:
But eventually we can but in a comment to show how this can be done and disable it by default?!?

Reply:
I would not test enforcement mode, since this would be testing the policy engine itself, which is already tested by the product team. I would focus on testing your custom policy. Like in this example passing in credentials is not required. If this changes, we can add this line, for now its not needed. Ok?

Reply:
Well, I don't test the enforcement mode (I assume this works), but as the testing takes quite a while, I test for some cases, that something isn't true  without the policy and true with the policy - so that the policy actually has an effect.